core/core.go

```diff
@@ -59,16 +59,17 @@ func (c *Core) InsertChain(blocks types.Blocks) (int, error) {
 	nodeCtx := common.NodeLocation.Context()
 	domWait := false
 	for i, block := range blocks {
-		isCoincident := c.sl.engine.IsDomCoincident(block.Header())
 
-		// Write the block body to the db.
-		rawdb.WritePendingBlockBody(c.sl.sliceDb, block.Header().Root(), block.Body())
+		isCoincident := c.sl.engine.IsDomCoincident(block.Header())
+		// Write the block body to the lru cache in worker.
```

Comment:
This comment is wrong. No cache here.

Reply:
updated the comment

Reply:
Where? It is still showing as of c52e9ac648f847bf23b370b6159a1529590abdcc

Reply:
check now